lib/src/editor/editor_utils.dart

```diff
@@ -426,8 +427,15 @@ class EditorConfig {
 
   /// Aspect ratio of crop rect
   /// default is custom
+  /// user can't change the ratio if this field is set
   final double? cropAspectRatio;
 
+  /// Initial Aspect ratio of crop rect
+  /// default is custom
+  /// This argument will only set the initial aspect ration and user can change it based on desire
+  /// despite of cropAspectRatio
```

Comment:
This is much better.
```suggestion
  /// Initial Aspect ratio of crop rect
  /// default is custom
  ///
  /// The argument only affects the initial aspect ratio,
  /// users can set it based on the desire despite of [cropAspectRatio].
```